features/step_definitions/project.rb

```diff
@@ -66,6 +66,19 @@
   end
 end
 
+# create a new project w/ a leading digit to test OVS quoting
+Given /^I create a project with leading digit name$/ do
```

Comment:
You could also unify with the non-leading step like:
`I create a project with (non-)?leading digit name`

but it is fine to leave as is